modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1579,6 +1579,186 @@ definition {
 		AppBuilderAdmin.validateIfSearchItemIsNotFound(itemName = "Apple");
 	}
 
+	@description = "Unpublish and Republish an App"
+	@priority = "5"
+	test UnpublishAndRepublish {
+		property portal.acceptance = "false";
```

Comment:
Hey @nkkwon , Thank you for the improvements! Could you please just remove the annotation related to the acceptance test? Thanks!